### ISSUE TITLE
App crashes with `AVAssetDownloadTask does not support response property` exception

### ISSUE THREAD

**Issue by Hans92** [Instabug/Instabug-iOS#496]
### Steps to Reproduce the Problem
Use `AVAssetDownloadTask` to download some media asset and try to cancel it

### Expected Behavior
App shouldn't crash

### Actual Behavior
App is crashing when calling `cancel()` on a `AVAssetDownloadTask` and throwing an exception `AVAssetDownloadTask does not support response property`

```
__74-[IBGSwizzler newImplementatioDataTaskWasCancelled:selector:swizzlerInfo:]_block_invoke_2
```
### Instabug integration code
```
Instabug.start(withToken: <TOKEN>, invocationEvents: [.screenshot])
Instabug.tintColor = UIColor(Theme.Color.black02)
Instabug.setColorTheme(.light)
Instabug.welcomeMessageMode = .disabled
Instabug.setReproStepsFor(.all, with: .enable)
Instabug.enabled = isEnabled
BugReporting.promptOptionsEnabledReportTypes = [.bug, .feedback]
BugReporting.bugReportingOptions = [.emailFieldHidden]
BugReporting.shouldCaptureViewHierarchy = true
BugReporting.enabled = isEnabled
Replies.enabled = false
```

### SDK Version
13.1.1
### iOS Version
17.5.1
### Device Model
iPhone 15 Pro Max

It looks like `IBGSwizzler` doesn't support `AVAssetDownloadTask`. I tried to disable NetworkLogger but it didn't help. When I remove Instabug integration code the app works properly.

**Comment by HGaurav328** [comment]
Also we are facing same issue. App crashing after calling cancel(). but after removing Instabug its working fine.

**Comment by AhmedAshraf605** [comment]
HI @Hans92 @HGaurav328 
This should be fixed in 13.3.0
